custom_components/bhyve/switch.py

```diff
@@ -396,6 +401,12 @@ def _setup(self, device):
         status = device.get("status", {})
         watering_status = status.get("watering_status")
 
+        next_start_programs = status.get("next_start_programs")
+
+        next_runtime = orbit_time_to_local_time(
+            status.get("next_start_time")
+        ).isoformat()
```

Comment:
`orbit_time_to_local_time` can return `None`, which will throw an exception when calling `isoformat()`. I'll fix this up.

Reply:
Awesome thanks